htdocs/core/class/html.formticket.class.php

```diff
@@ -1567,7 +1567,7 @@ public function showMessageForm($width = '40%')
 			if (empty($topic)) {
 				print '<td><input type="text" class="text minwidth500" name="subject" value="['.getDolGlobalString('MAIN_INFO_SOCIETE_NOM').' - '.$langs->trans("Ticket").' '.$ticketstat->ref.'] '. $ticketstat->subject .'" />';
 			} else {
-				print '<td><input type="text" class="text minwidth500" name="subject" value="['.getDolGlobalString('MAIN_INFO_SOCIETE_NOM').' - '.$langs->trans("Ticket").' '.$ticketstat->ref.'] '.$topic.'" />';
+				print '<td><input type="text" class="text minwidth500" name="subject" value="'.make_substitutions($topic, $this->substit).'" />';
```

Comment:
Why removing the prefix 
[Thecompany - Ticket 123] ...
in default email topic ?

Reply:
If the user has defined a subject by default, he can add it to the templates himself; on the other hand, if he has left it in, he can't remove it, which can be painful for the user experience.